.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Rotate `CHANGELOG.md` (following [Keep a Changelog](https://keepachangelog.com/) principles)
+- [ ] Ensure outstanding changes are committed:
+```bash
+$ git add .
+$ git commit -m "X.Y.Z"
+```
+- [ ] Publish the release branch:
+```bash
+$ git flow release publish X.Y.Z
+```
+- [ ] Test the release branch on staging
+    - Wait for CI to cycle the release branch onto staging
+    - Thoroughly vet new features
+    - Commit any last-minute fixes
+- [ ] Start a new [release pipeline job](http://civicci01.internal.azavea.com/view/oar/job/Open%20Apparel%20Registry%20Release%20Pipeline/build?delay=0sec) with the SHA of `release/X.Y.Z` that was tested on staging
+- [ ] Run migrations, if applicable (see the `showmigrations` release pipeline stage):
+```bash
+$ ./scripts/manage.py ecsmanage -e production migrate
+```
+- [ ] Finish and publish the release branch:
+```bash
+$ git flow release finish -p X.Y.Z 
```

Comment:
`finish` prompts you to add merge commit messages. On OTM the convention was to just put the release number in as the commit message. We should consider suggesting that convention, or some other convention to keep things consistent.

Reply:
When I cut releases, I leave the merge commit message as-is, and set the tag message to `X.Y.Z`. Any thoughts on this?

Reply:
👍
I am not familiar with `-p`. What does that do? Had trouble finding documentation.

Reply:
My understanding is it runs the equivalent of:

```bash
$ git flow release finish X.Y.Z
$ git push origin develop
$ git checkout master
$ git push origin master
$ git push --tags
```

See:

- https://github.com/petervanderdoes/gitflow-avh/wiki/Reference:-git-flow-release#git-flow-release-finish---finish-a-release-branch
- https://github.com/petervanderdoes/gitflow-avh/blob/641076945ff948b0da32405ca85f9d989faa6df2/git-flow-release#L177

Reply:
Ah. It's an AVH thing. Thanks
👍

Reply:
It looks like it's in the regular one too:

https://github.com/nvie/gitflow/wiki/Command-Line-Arguments#git-flow-release-finish--fsumpkn-version

BTW, is AVH non-standard? I've been using it, but I realize that this other repo has many more stars.

Reply:
🤦‍♂️ I just missed it
I believe we standardized on AVH because it had more features. That was a while ago.